src/views/Home/index.tsx

```diff
@@ -64,6 +66,13 @@ const Home: React.FC = () => {
             <UserBanner />
           </UserBannerWrapper>
         )}
+        <Flex
+          pt={[account ? '220px' : '0', null, null, account ? '76px' : '0']}
+          mt={[account ? '0' : '-16px', null, null, account ? '0' : '-48px']}
```

Comment:
I won't even ask.

Reply:
This is just revert of what was previously there 😄  I think Hutch did the banner 👀

Reply:
hahahahha.

Basically the calcs to push everything else on the home page below the absolute-positioned user banner.